Gemfile.lock

```diff
@@ -62,18 +60,15 @@ GEM
     rexml (3.2.6)
     rouge (4.1.3)
     safe_yaml (1.0.5)
-    sass-embedded (1.64.2)
+    sass-embedded (1.64.1-arm64-darwin)
       google-protobuf (~> 3.23)
-      rake (>= 13.0.0)
     terminal-table (3.0.2)
       unicode-display_width (>= 1.1.1, < 3)
     unicode-display_width (2.4.2)
     webrick (1.8.1)
 
 PLATFORMS
   arm64-darwin-22
-  x86_64-darwin-19
-  x86_64-linux
```

Comment:
This will break CI/deploys (see: https://github.com/GlareDB/glaredb.github.io/issues/125)

Reply:
@greyscaled  I just reverted the gemfile.lock, should be good to go now.